lib/cisco_node_utils/cmd_ref/radius_server.yaml

```diff
@@ -1,66 +1,96 @@
 # radius_server.yaml
 ---
-_exclude: [ios_xr]
+_template:
+  nexus:
+    get_command: "show running-config radius all"
+  ios_xr:
+    get_command: "show running-config radius-server"
+    multiple: true
 
 accounting:
+  _exclude: [ios_xr]
   auto_default: false
   kind: boolean
   get_command: "show running-config radius"
-  get_value: '/^radius-server host %s.* (accounting) .*$/'
+  get_value: '/^radius-server host <ip>.* (accounting) .*$/'
   set_value: '<state> radius-server host <ip> accounting'
   default_value: true
 
 acct-port:
-  kind: int
-  get_command: "show running-config radius all"
-  get_value: '/^radius-server host %s.* acct-port (\d+).*$/'
-  set_value: '<state> radius-server host <ip> acct-port <port>'
-  default_value: 1813
+  get_value: '/^radius-server host <ip>.* acct-port (\d+).*$/'
+  nexus:
+    default_value: 1813
+    set_value: '<state> radius-server host <ip> acct-port <port>'
+  ios_xr:
+    default_value: 1646
 
 auth-port:
-  kind: int
-  get_command: "show running-config radius all"
-  get_value: '/^radius-server host %s.* auth-port (\d+).*$/'
-  set_value: '<state> radius-server host <ip> auth-port <port>'
-  default_value: 1812
+  get_value: '/^radius-server host <ip>.* auth-port (\d+).*$/'
+  nexus:
+    default_value: 1812
+    set_value: '<state> radius-server host <ip> auth-port <port>'
+  ios_xr:
+    default_value: 1645
 
 authentication:
+  _exclude: [ios_xr]
   auto_default: false
   kind: boolean
   get_command: "show running-config radius"
-  get_value: '/^radius-server host %s.* (authentication) .*$/'
+  get_value: '/^radius-server host <ip>.* (authentication) .*$/'
   set_value: '<state> radius-server host <ip> authentication'
   default_value: true
 
-hosts:
+host_port_pairs:
+  _exclude: [nexus]
   multiple: true
-  get_command: "show running-config radius"
+  get_command: "show running-config radius-server"
+  get_value: '/^radius-server host <ip> auth-port (\d+) acct-port (\d+)/'
+  default_value: ''
+
+hosts:
```

Comment:
`hosts` is no longer defined as `multiple` for Nexus, if I'm reading this right. Doesn't this break `RadiusServer.radiusservers` on Nexus?